client-index/src/tendermint/types/block.rs

```diff
@@ -37,20 +37,27 @@ impl Block {
     }
 }
 
+// Note: Do not change these values. These are tied with tests for `RpcSledIndex`
 #[cfg(test)]
-mod tests {
-    use super::*;
-
-    #[test]
-    fn check_transactions() {
-        let block = Block {
+impl Default for Block {
+    fn default() -> Self {
+        Block {
             block: BlockInner {
                 data: Data {
                     txs: vec!["+JWA+Erj4qBySKi4J+krjuZi++QuAnQITDv9YzjXV0RcDuk+S7pMeIDh4NaAlHkGYaL9naP+5TyquAhZ7K4SWiCliAAA6IkEI8eKw4GrwPhG+ESAAbhASZdu2rJI4Et7q93KedoEsTVFUOCPt8nyY0pGOqixhI4TvORYPVFmJiG+Lsr6L1wmwBLIwxJenWTyKZ8rKrwfkg==".to_owned()]
                 }
             }
-        };
+        }
+    }
+}
 
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn check_transactions() {
```

Comment:
Why exactly are here these tests of `Default` implementation?

Reply:
This is testing `transactions()` with valid `Block` data. `Default` implementation just provides a valid `Block` data.

Other reason for `Default` implementation is that these are used in `MockClient`. The need of these `Default` implementations will go away once we have a way to test client with actual tendermint node (integration tests).

Reply:
The values in these `Default` implementations are coupled with tests in `RpcSledIndex` where we transfer some coins from one address to other address and check their balances and transaction history.